docker-compose-dev.yml

```diff
@@ -77,6 +81,10 @@ services:
       POSTGRES_PASSWORD: us
     volumes:
       - ./metadata:/var/lib/postgresql/data
+ 
+    ports:
+      - "8086:5432"
+
 
   us:
     container_name: datalens-us
```

Comment:
Don't forget to update container versions for ui and us before merging

Reply:
yes, will release ui and us first, then update yml with released versions